catalog/resource_grants.go

```diff
@@ -93,16 +94,27 @@ type securableMapping map[string]map[string]bool
 
 // reuse ResourceDiff and ResourceData
 type attributeGetter interface {
-	Get(key string) any
+	GetRawConfig() cty.Value
 }
 
 func (sm securableMapping) kv(d attributeGetter) (string, string) {
+	rawConfig := d.GetRawConfig()
+	if rawConfig.IsNull() {
+		return "unknown", "unknown"
```

Comment:
We should return some kind of error here. We basically don't expect this case in the happy path. If the user hasn't specified any resource, we should say so ("Specify one of: ..."). If they have, it is definitely a provider bug.

Reply:
the way we define the schema of the resource means tf will fail if the user does not specify one of the securables